### PR TITLE
Link tasks to users

### LINKED ISSUE #1: Link tasks to users

### PR DESCRIPTION
**Pull request by nicordev**
Will close #1 #5

### CODE REVIEW

File: src/AppBundle/Security/TaskVoter.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+use AppBundle\Entity\Task;
+use AppBundle\Entity\User;
+use Symfony\Component\Security\Core\Authentication\Token\TokenInterface;
+use Symfony\Component\Security\Core\Authorization\Voter\Voter;
+
+class TaskVoter extends Voter
+{
+    public const EDIT = "edit";
+    public const DELETE = "edit";
```
Comment (olix21) [inline src/AppBundle/Security/TaskVoter.php]: "remove"

File: src/AppBundle/Security/TaskVoter.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    protected function voteOnAttribute($attribute, $subject, TokenInterface $token)
+    {
+        $user = $token->getUser();
+
+        if (!$user instanceof User) {
+            return false;
+        }
+
+        if (in_array($attribute, [self::EDIT, self::DELETE])) {
+            if (!$subject instanceof Task) {
```
Comment (olix21) [inline src/AppBundle/Security/TaskVoter.php]: Should be moved in the `supports` method. 
This voter can only vote for `Task` and should not support another entity

**Review by olix21 — CHANGES_REQUESTED** [review]

File: src/AppBundle/Security/TaskVoter.php (hunk 1 above)
Comment (nicordev) [reply src/AppBundle/Security/TaskVoter.php]: Done.

File: src/AppBundle/Security/TaskVoter.php (hunk 2 above)
Comment (nicordev) [reply src/AppBundle/Security/TaskVoter.php]: Done.

**Review by olix21 — APPROVED** [review]

**Pull request merged**